src/driver/test_point_generator.py

```diff
@@ -37,6 +37,12 @@
 # give an acceptably small sampling variance for the modeling errors.
 MINIMUM_NUMBER_OF_TEST_POINTS = 100
```

Comment:
Would you remove this since you added MINIMUM_NUMBER_OF_TEST_POINTS_PER_PUBLISHER